annotation-file-utilities/src/annotator/Main.java

```diff
@@ -625,6 +630,12 @@ public int compare(Insertion i1, Insertion i2) {
           parsedSpec = filtered;
           verb.debug("After filtering: %d annotations from %s%n", parsedSpec.size(), jaifFile);
         }
+        // if (dbug.isEnabled()) {
```

Comment:
Should this be deleted?

Reply:
Thanks for checking.  I'm going to leave it because it may be useful for debugging later.  If it is too much clutter, it can be deleted later.